docs/user_guide/HOWTO/IPA_Clients.rst

```diff
@@ -74,29 +77,43 @@ To be able to add hosts from the command line:
       # In <domain>.yaml or whichever level is most specific
       ---
       classes:
-      - simp::ipa::install
-      - simp::ipa::client
+      - simp_ipa::client::install
 
-      simp::ipa::install::ensure: present
-      simp::ipa::install::password: "%{trusted.certname}-<password from script above>"
-      simp::ipa::install::server: ipa.example.domain
-      simp::ipa::install::domain: example.domain
-      simp::ipa::install::realm: EXAMPLE.DOMAIN
+      simp_ipa::client::install::ensure: present
+      simp_ipa::client::install::password: "%{trusted.certname}-<password from script above>"
+      simp_ipa::client::install::server: ipa.example.domain
+      simp_ipa::client::install::domain: example.domain
+      simp_ipa::client::install::realm: EXAMPLE.DOMAIN
 
+      # This setting should already exist, it needs to be modified
```

Comment:
Don't have to add this yourself.  sssd class does it for you automatically once you are joined to a domain.  However, this does mean more than 1 puppet run may be required to be completely configured.